package.json

```diff
@@ -42,6 +42,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "binary-search": "^1.3.2"
+    "binary-search": "^1.3.2",
+    "randombytes": "^2.0.3"
```

Comment:
Great idea, i think the browser package size reduction is definitely worth the extra dependency in this case.